code/modules/reagents/chemistry/holder.dm

```diff
@@ -350,11 +350,11 @@
 					R.on_mob_metabolize(C)
 				if(can_overdose)
 					if(R.overdose_threshold)
-						if(R.volume >= R.overdose_threshold && !R.overdosed)
+						if(R.volume > R.overdose_threshold && !R.overdosed) //This is a certified AUStation code moment, PR: #3045. Changed from >= to >
```

Comment:
Made me chuckle but you should probably follow the standard way to do it.

`//austation -- [comment]`